kuma/static/styles/minimalist/components/_breadcrumbs.scss

```diff
@@ -17,17 +19,65 @@
             }
         }
 
-        .breadcrumb-chevron:after {
-            display: inline-block;
-            content: '›';
-            color: $grey-vdark;
-            margin: 0 5px;
-            font-size: 1rem;
-            font-weight: bold;
+        .breadcrumb-previous {
+            &::before {
+                @include caret-arrow($blue, left);
+                margin: 0 10px 0 0;
+                position: relative;
+                top: -1px;
+            }
+            span::before {
+                display: inline-block;
+                content: 'See\00a0';
```

Comment:
I am not entirely sure why we have this. I know it is per design but, it seems a little odd to me. for one thing, if this is important to provide context, we should not add this via generated content as it will not be accessible[1].

If it is not important, I would vote to not include it.

[1] https://developer.mozilla.org/en-US/docs/Web/CSS/content#Accessibility_concerns

Reply:
@malqinneh Thoughts?

Reply:
It's to communicate to users that the breadcrumb label is meant for a section that they are *not* currently on. If that makes sense.

Reply:
In that case, I can move the "See " text into the JSX file and hide it for desktop.

Reply:
Thank you Mindy and Mustafa